includes/CLI/Plugin_Check_Command.php

```diff
@@ -95,9 +95,12 @@ public function __construct( Plugin_Context $plugin_context ) {
 	 * : Include experimental checks.
 	 *
 	 * [--exclude-directories=<directories>]
-	 * : Additional directories to exclude from checks
+	 * : Additional directories to exclude from checks.
 	 * By default, `.git`, `vendor` and `node_modules` directories are excluded.
 	 *
+	 * [--exclude-files=<files>]
+	 * : Additional files to exclude from checks.
+	 *
```

Comment:
Not to self: at some point in the future we should just make this one `--exclude` arg for better DX.